ecommerce/settings/base.py

```diff
@@ -834,3 +834,7 @@
 # To check government purchase restriction lists
 SDN_CHECK_API_URL ="https://api.trade.gov/gateway/v1/consolidated_screening_list/search"
 SDN_CHECK_API_KEY = "sdn search key here"
+
+# Enable and disable tax rates
+ENABLE_TAX_RATE = True
+TAX_RATE = '0.3'
```

Comment:
@giovannicimolin, these can and should be defined via Ansible configuration (`ECOMMERCE_EXTRA_CONFIG_OVERRIDES`).

Reply:
@lgp171188 There's no need for defining variables there, we can leave sane defaults here (`TAX_RATE = '0.0'` and maybe remove the `ENABLE_TAX_RATE` variable) and use `ECOMMERCE_EXTRA_CONFIG_OVERRIDES` to change this during configuration.

Values on `ECOMMERCE_EXTRA_CONFIG_OVERRIDES` will automatically override any configuration variable as long as they are defined in `ecommerce.yml`.